_posts/2022-12-05-neptune-ai-integration-released.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+author: Hamza Tahir
+title: "Tracking experiments in your MLOps pipelines with ZenML and Neptune"
+description: "ZenML 0.23.0 comes with a brand-new experiment tracker flavor - Neptune.ai! We dive deeper in this blog post."
+category: zenml
+tags: zenml release-notes
+publish_date: December 05th, 2022
+date: 2022-12-05T00:02:00Z
+thumbnail: /assets/posts/neptune_released/Release_0.23.0.gif
+image:
+  path: /assets/posts/neptune_released/Release_0.23.0.jpg
+---
+
+**Last updated:** December 05, 2022.
+
+![Release 0.23.0](../assets/posts/neptune_released/Release_0.23.0.jpg)
+
+ZenML 0.23.0 was released this week, and comes bundled with a brand new `neptune` 
+experiment tracker flavor. [Neptune](https://neptune.ai/product/experiment-tracking) is a
+popular platform that you would normally use in the iterative ML experimentation
+phase to track and visualize experiment results. Neptune can also
+track and visualize the results produced by your
+ZenML pipeline runs, as you make the transition towards a more production
+oriented workflow.
+
+A big shout-out to [@AleksanderWWW](https://github.com/AleksanderWWW), from the Neptune team
+who spearheaded the [Neptune integration](https://zenml.io/integrations/neptune)
+with [#1082](https://github.com/zenml-io/zenml/pull/1082).
+
+## :couple_with_heart_man_man: How Neptune fits with ZenML
```

Comment:
I'd probably change the emoji to something neutral so we don't send the wrong message to the public 😂